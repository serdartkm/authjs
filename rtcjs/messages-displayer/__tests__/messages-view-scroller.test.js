import React from 'react'
import MessagesViewScroller from '../lib/messages-view-scroller'
import renderer from 'react-test-renderer'



describe('MessagesViewScroller', () => {
    it("renders correctly", () => {
        const tree = renderer.create(<MessagesViewScroller></MessagesViewScroller>)
        expect(tree).toMatchSnapshot();
    })
})