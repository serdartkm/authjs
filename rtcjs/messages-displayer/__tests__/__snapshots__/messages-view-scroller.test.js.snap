// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessagesViewScroller renders correctly 1`] = `
<div
  name="msgViewScroller"
  style={
    Object {
      "backgroundColor": "#edeff2",
      "flex": 10,
      "overflow": "auto",
      "width": "100%",
    }
  }
/>
`;
